.github/workflows/deploy-to-heroku.yml

```
@@ -1,16 +1,26 @@
+name: Deploy to Heroku
+
 on:
   push:
     branches:
       - master
+  # Deploys to Heroku once each day
+  schedule:
+    - cron: "0 0 * * *"
 
 jobs:
   build:
     runs-on: ubuntu-latest
     environment: balena-docs
     steps:
-      - uses: actions/checkout@v2
-      - uses: akhileshns/heroku-deploy@v3.12.12 # This is the action
+      # Checks out a copy of your repository on the ubuntu-latest machine
+      - name: Checkout code
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v3
```
v3 came out recently with no interface changes, might as well use it

---

Thanks for noting this @klutchell I will slip it in a PR later